.travis.yml

```diff
@@ -0,0 +1,98 @@
+#=============================================================================
+# Travis CoreNeuron settings
+#=============================================================================
+
+#=============================================================================
+# Environment
+#=============================================================================
+# Use new Travis infrastructure (Docker can't sudo yet)
+sudo: false
+
+#=============================================================================
+# Build matrix
+#=============================================================================
+matrix:
+  fast_finish: true
+  include:
+    - language: cpp
+      os: linux
+      dist: xenial
+      env:
+        - MATRIX_EVAL="CC=mpicc && CXX=mpicxx"
+#        - PYTHON_VERSION=3.6.7
+    - language: cpp
+      os: osx
+      osx_image: xcode10.2
+      env:
+        - MATRIX_EVAL="CC=mpicc && CXX=mpicxx"
+
+#=============================================================================
+# Common Packages
+#=============================================================================
+addons:
+  # for Linux builds, we use APT
+  apt:
+    sources:
+      - ubuntu-toolchain-r-test
+    packages:
+      - flex
+      - bison
+      - cmake
+      - mpich
```

Comment:
IIRC, this should be `libmpich-dev` (or `libopenmpi-dev`) ?

Reply:
Yeap.. @WeinaJi please check what are exactly the package names on the respective system (you should be able to look this up on your workstation for example.